_sass/components/_news-section.scss

```diff
@@ -0,0 +1,102 @@
+.c-latest-news {
+  width: 80%;
+  margin: 4rem auto 3.5rem;
+
+  @media (min-width: $bp-md) {
+    margin: 8rem auto 7.5rem;
+  }
+
+  &__title-container {
+    @include display-flex-news;
+
+    @media (min-width:$bp-md) {
```

Comment:
Can we add a space after the colon?

Reply:
I'm not seeing a space between `min-width:` and `$bp-md`

Reply:
oops